src/ray/core_worker/test/BUILD.bazel

```diff
@@ -29,8 +29,29 @@ ray_cc_test(
 )
 
 ray_cc_test(
-    name = "direct_actor_transport_test",
-    srcs = ["direct_actor_transport_test.cc"],
+    name = "task_receiver_test",
```

Comment:
Split `direct_actor_transport_test` into `task_receiver_test` and `actor_task_submitter_test`.

Reply:
nice!